cibuildwheel/linux.py

```diff
@@ -12,6 +13,26 @@
 )
 
 
+def matches_platform(identifier):
+    pm = platform.machine()
+    if pm == "x86_64":
+        if identifier.endswith('x86_64') or identifier.endswith('i686'):
```

Comment:
Maybe this is this worth a comment, like 'x86_64 machines can run i686 docker containers`?

Reply:
Done in: https://github.com/joerick/cibuildwheel/pull/273/commits/05df9bbfd413122bf6fbda0e317abfe882290416